tax/extensions.go

```diff
@@ -338,6 +338,42 @@ func (v validateExtCodeValues) Validate(value interface{}) error {
 	return nil
 }
 
+// ExtensionsExcludeCodes returns a validation rule that ensures the extension map's
+// key does not have any of the provided **codes**.
+func ExtensionsExcludeCodes(key cbc.Key, codes ...cbc.Code) validation.Rule {
+	return validateExtCodeExcludeValues{
+		key:    key,
+		values: codes,
+	}
+}
+
+type validateExtCodeExcludeValues struct {
+	key    cbc.Key
+	values []cbc.Code
+}
```

Comment:
Rather than creating a completely new structure, it'd be more efficient to modify the existing `validateExtCodeIncludesValues` (or whatever name is), to be more generic: `validateExtCodes` and apply flags.